src/plugins/commit-message/index.js

```diff
@@ -6,6 +6,8 @@
 
 "use strict";
 
+const utils = require("../../utils");
```

Comment:
nit: You can use destruction here to just grab the util function you are using.

Reply:
Is that preferred in this project? In ESLint we like to import all of lodash and qualify the functions in code, rather than destructuring or getting at the property, so that's what I did here.

Reply:
This project is based on > node 8 and ESLint is based on > Node 4. Thats the reason.
Thats is why i started to use airbnb guide so that we can respect modern syntax and our style guide does not.